t3/simulate/rms_constantV_simulator.py

```diff
@@ -0,0 +1,217 @@
+"""
+RMS Simulator Adapter module
+Used to run mechanism analysis with RMS that simulates ideal gases at constant V
```

Comment:
From talking with Matt, these are solved adiabatically since there's no heat transfer in or out of the domain. I think adiabatic is a more accurate description for constantV? I too want the description to be accurate and maybe I'm confusing myself

Reply:
Take a look at NASA CEA (a widely used software to predict products of fuels based on thermodynamics only). Look at "type code" in the from page. E.g., you have `hp` for Assigned Enthalpy & Pressure (so what we'll call `H-P`), and so on. We definitely don't have to conform to that, but it does reflect on the kinds of problems we care about. Constant V should go with constant U, so we can say "Constant V-U (adiabatic)".

Reply:
I don't want to be obnoxious or pedantic, but we should conform to UV and HP (constant (internal) energy/constant volume & constant enthalpy/constant pressure) and while it's possible to pick a different pair of thermodynamic properties to hold constant, these two cases are drawn directly from the first law of thermodynamics and are essential standards.